Documentation/concepts/ebpf/lifeofapacket.rst

```diff
@@ -0,0 +1,100 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+################
+Life of a Packet
+################
```

Comment:
Unless I am mistaken, all the contents you add in this file and the other ones created in this commit come as duplicates with the `concepts/ebpf/intro.rst` (formerly `architecture.rst`). I think you intended to move it but forgot to remove the original text?